CHANGELOG.md

```diff
@@ -1,170 +1,17 @@
 # Change log
 
-The `next` version of 4.20 is now available.  Planned release date is June 2021.
+The `next` version of 4.21 is now available.  Planned release date is September 2021.
 
 ![Current build version](https://img.shields.io/npm/v/arcgis-js-api/next?label=Current%20build)
 
 ## Breaking changes
 
-* CodedValueDomain.codedValues now returns an array of CodedValue type definition objects. Prior to this, it used an array of generic objects. The properties within the codedValue have not changed.
-* RotateEventInfo.angle is now reported in degree by the SketchViewModel.
-* The keyboard shortcut key has changed from Ctrl to Shift in Sketch and SketchViewModel for creating a polgyon graphic (rectangle or circle) with predefined shapes.
-* The keyboard shortcut key has changed from C to Enter in Sketch, SketchViewModel, and Draw for completing a polyline, polygon, or point graphic.
-
-The following classes, methods, properties and events have been deprecated for at least 2 releases and have now been removed from the API:
-
-| Class/Property/Method/Event | Alternate option | Version deprecated |
-|----------|-------------|--------------------|
-|Bookmarks.select-bookmark|Bookmarks.bookmark-select|4.17|
-|BookmarksViewModel.BookmarkOptions.captureExtent|BookmarksViewModel.BookmarkOptions.captureViewpoint|4.17|
-|FeatureTemplates.filterEnabled|FeatureTemplates.visibleElements.filter|4.15|
-|Popup.featureNavigationEnabled|Popup.visibleElements.featureNavigation|4.15|
+TBD
```

Comment:
instead of TBD - you could add the one breaking change that's already there: 

> <p style="margin: 1rem 0px; padding: 0px; box-sizing: border-box; color: rgb(21, 21, 21); font-size: 16px; line-height: 1.66; font-family: &quot;Avenir Next W01&quot;, &quot;Avenir Next W00&quot;, &quot;Avenir Next&quot;, Avenir, &quot;Helvetica Neue&quot;, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: 0.1px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">The following classes, methods, properties and events have been deprecated for at least 2 releases and have now been removed from the API:</p>
> 
> Class/Property/Method/Event | Alternate option | Version deprecated
> -- | -- | --
> Bookmarks.bookmarkCreationOptions | Bookmarks.defaultCreateOptions | 4.18
> 
> <p style="margin: 1rem 0px; padding: 0px; box-sizing: border-box; color: rgb(21, 21, 21); font-size: 16px; line-height: 1.66; font-family: &quot;Avenir Next W01&quot;, &quot;Avenir Next W00&quot;, &quot;Avenir Next&quot;, Avenir, &quot;Helvetica Neue&quot;, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: 0.1px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Please refer to the<span> </span><a href="https://local.arcgis.com:8000/breaking-changes/" style="margin: 0px; padding: 0px; box-sizing: border-box; background-color: transparent; color: var(--citra-link-purple-1); text-decoration: none; text-shadow: 0.03em 0 var(--afd-flow-content-background), -0.03em 0 var(--afd-flow-content-background), 0 0.03em var(--afd-flow-content-background), 0 -0.03em var(--afd-flow-content-background), 0.06em 0 var(--afd-flow-content-background), -0.06em 0 var(--afd-flow-content-background), 0.09em 0 var(--afd-flow-content-background), -0.09em 0 var(--afd-flow-content-background), 0.12em 0 var(--afd-flow-content-background), -0.12em 0 var(--afd-flow-content-background), 0.15em 0 var(--afd-flow-content-background), -0.15em 0 var(--afd-flow-content-background); display: inline; background-image: linear-gradient(var(--citra-link-purple-1), var(--citra-link-purple-1)); background-size: 1px 1px; background-repeat: repeat-x; background-position: 0% 1.33em; overflow-wrap: break-word;">Breaking changes</a><span> </span>guide topic for a complete list of breaking changes across all releases of the 4x API.</p>